libs/testing/src/jobs.service.ts

```diff
@@ -80,6 +80,35 @@ export class JobsService {
         },
       });
     } while (totalCount > 0 || runningJobs > unfinishedJobs);
+
+    return {
+      getDelayedTimestamp: async () => {
+        const delayedJobs = await this.standardQueue.getDelayed();
+
+        if (delayedJobs.length === 1) {
+          return delayedJobs[0].delay;
+        } else {
+          if (delayedJobs.length > 1) {
+            throw new Error('There are more than one delayed jobs');
+          } else if (delayedJobs.length === 0) {
+            throw new Error('There are no delayed jobs');
+          }
+        }
+      },
+      runDelayedImmediatly: async () => {
```

Comment:
This new testing functionality allows running immediatly delayed jobs on bull mq